MVP Module/Source/Feed Presentation/FeedPresenter.swift

```diff
@@ -9,21 +9,28 @@ protocol FeedLoadingView {
 	func display(_ viewModel: FeedLoadingViewModel)
 }
 
+protocol FeedErrorView {
+	func display(_ viewModel: FeedErrorViewModel)
+}
+
 protocol FeedView {
 	func display(_ viewModel: FeedViewModel)
 }
 
+typealias FeedViewFlow = FeedView & FeedErrorView
+
 final class FeedPresenter {
-	private let feedView: FeedView
+	private let feedView: FeedViewFlow
 	private let loadingView: FeedLoadingView
 
-	init(feedView: FeedView, loadingView: FeedLoadingView) {
+	init(feedView: FeedViewFlow, loadingView: FeedLoadingView) {
 		self.feedView = feedView
 		self.loadingView = loadingView
 	}
 
 	func didStartLoadingFeed() {
 		loadingView.display(FeedLoadingViewModel(isLoading: true))
+		feedView.display(FeedErrorViewModel(errorMessage: nil))
```

Comment:
And we can change accordingly on which object is responsible for displaying/hiding errors:
```suggestion
		errorView.display(FeedErrorViewModel(errorMessage: nil))
```

Reply:
done